server/network.js

```diff
@@ -312,12 +311,7 @@ network.getIncidentEdges_ = function(file, gene) {
  */
 network.getComb_ = function(file, exp) {
```

Comment:
Besides if the combined regulation is not working properly please fix in a separate PR.